org.kde.kdeconnect.json

```diff
@@ -0,0 +1,83 @@
+{
+    "id": "org.kde.kdeconnect",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "kdeconnect-indicator",
+    "rename-icon": "kdeconnect",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+
+        "--own-name=org.kde.kdeconnectd",
+        "--own-name=org.kde.kdeconnect-indicator",
+
+        "--talk-name=org.mpris.MediaPlayer2",
+        "--talk-name=org.mpris.MediaPlayer2.Player"
+    ],
+    "rename-desktop-file": "org.kde.kdeconnect.nonplasma.desktop",
+    "rename-icon": "kdeconnect",
+    "cleanup": [
+        "/include",
+		"/lib/pkgconfig",
+		"/lib/*.la",
+		"/lib/cmake",
+		"/share/doc",
+		"/share/man"
+    ],
+    "modules": [
+        {
+            "name": "libfakekey",
```

Comment:
this needs to be reconfigured to work on aarch64, similar to https://github.com/flathub/fr.free.Homebank/blob/91c81b69cfe98bfea5593ba6475c3b69b7308d04/fr.free.Homebank.json#L58